Source/WebCore/platform/mediastream/libwebrtc/gstreamer/GStreamerVideoEncoderFactory.cpp

```diff
@@ -50,7 +50,9 @@
 #include <wtf/text/StringConcatenateNumbers.h>
 
 GST_DEBUG_CATEGORY(webkit_webrtcenc_debug);
+#if !defined(GST_CAT_DEFAULT)
```

Comment:
The right fix here would be to flag this file as `@no-unify`.

Reply:
Or maybe some other file is not undeffing GST_CAT_DEFAULT at the end of its code. I can take a look.

Reply:
https://bugs.webkit.org/show_bug.cgi?id=258514

Reply:
Thank you!